src/test/java/com/microsoft/sqlserver/jdbc/connection/ReplicationTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Microsoft JDBC Driver for SQL Server Copyright(c) Microsoft Corporation All rights reserved. This program is made
+ * available under the terms of the MIT License. See the LICENSE file in the project root for more information.
+ */
+package com.microsoft.sqlserver.jdbc.connection;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+
+import com.microsoft.sqlserver.jdbc.RandomUtil;
+import com.microsoft.sqlserver.jdbc.SQLServerDataSource;
+import com.microsoft.sqlserver.jdbc.TestUtils;
+import com.microsoft.sqlserver.testframework.AbstractTest;
+import com.microsoft.sqlserver.testframework.Constants;
+import com.microsoft.sqlserver.testframework.DBTable;
+
+
+/*
+ * This test is for testing the replication connection property
+ */
+@RunWith(JUnitPlatform.class)
+@Tag(Constants.xAzureSQLDW)
+public class ReplicationTest extends AbstractTest {
+
+    @Test
+    public void testReplication() throws SQLException {
+        String tableName = RandomUtil.getIdentifier("repl");
+        String triggerName = RandomUtil.getIdentifier("trig");
+        String escapedTableName = DBTable.escapeIdentifier(tableName);
+        String escapedTriggerName = DBTable.escapeIdentifier(triggerName);

```

Comment:
don't use DBTable, that is old framework code that we are moving away from

Reply:
Is there a current utility method that replaces it?

Reply:
oh just use AbstractSQLGenerator.escapeIdentifier(tableName). As an example, see SQLServerConnectTest.java